CHANGELOG.md

```diff
@@ -10,9 +10,20 @@ Note that Sockeye has checks in place to not translate with an old model that wa
 
 Each version section may have have subsections for: _Added_, _Changed_, _Removed_, _Deprecated_, and _Fixed_.
 
+## [1.18.11]
+### Added
+- Introducing the image captioning module. Type of models supported: ConvNet encoder - Sockeye NMT decoders
```

Comment:
It might make sense to have a single bullet point for the image captioning module and then use sub-level bullet points for the details.

Reply:
Ok. I will probably remove most of the details and have a concise version of this.